src/test/java/com/google/cloud/tools/libraries/LibrariesTest.java

```diff
@@ -170,7 +170,8 @@ public void testVersionExists() throws IOException {
               + "/"
               + coordinates.getString("artifactId")
               + "/"
-              + coordinates.getString("version");
+              + coordinates.getString("version")
+              + "/";
```

Comment:
Would it make sense to change `assertReachable()` to call `setFollowRedirects(true)` on the HTTP connection?  Probably not — we want these URLs to be canonical.

Reply:
I was getting 404s for some reason, not 300's